src/pymor/models/examples.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of the pyMOR project (https://www.pymor.org).
+# Copyright pyMOR developers and contributors. All rights reserved.
+# License: BSD 2-Clause License (https://opensource.org/licenses/BSD-2-Clause)
+
+def thermal_block_example():
```

Comment:
Do we want to parametrize this or is it intentional that there are no parameters?

Reply:
There can be parameters. I just didn't want to decide what to parameterize.